molecule/default/tests/test_openstack_services.py

```diff
@@ -2,75 +2,121 @@
 # ==============================================================================
 # Imports
 # ==============================================================================
+import pytest_rpc.helpers as helpers
+import os
 import pytest
-from pprint import pformat
-from munch import unmunchify
+import testinfra.utils.ansible_runner
+import json
 
+# ==============================================================================
+# Globals
+# ==============================================================================
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('shared-infra_hosts')[:1]
+
+
+# ==============================================================================
+# Helpers
+# ==============================================================================
+def service_enabled(service, run_on_host):
+    """ Verify if a service is enabled
+
+    Args:
+        service (str): an openstack service to query (e.g 'nova', 'glance')
+        run_on_host (testinfra.host.Host): Testinfra host fixture
+
+    Return:
+        bool: whether the queried service is enabled
+
+    """
+
+    cmd = ". openrc ; openstack service show {} -f json".format(service)
+    output = helpers.run_on_container(cmd, 'utility', run_on_host)
+
+    try:
+        result = json.loads(output.stdout)
+
+    except ValueError as e:
+        result = str(e)
+
+    if result['enabled']:
```

Comment:
It looks like this line would result in a `TypeError` if the previous `ValueError` set the result to the error string.

For this helper, since you are only interested in one field of the service, it might be simpler to not use the json format.

Suggestion.
```
cmd = ". openrc ; openstack service show -f value -c enabled {}".format(service)
output = helpers.run_on_container(cmd, 'utility', run_on_host)
result = output.stdout
if result.lower() == 'true':
    return True
esle:
    return False
```

Reply:
Tested it with python 2.7 and it worked, however, I like your simpler solution of not using json when I only need one field, will update in the next push